gradle-plugins/plugin-base/src/main/java/io/spine/gradle/SpinePlugin.java

```diff
@@ -158,6 +168,53 @@ public Builder insertAfterTask(TaskName target) {
             }
 
             /**
+             * Specify tasks which will precede the new one.
+             *
+             * <p>Unlike {@link #insertAfterTask insertAfterTask()}, this method will depend
+             * the new task on <b>every</b> task with such name in the project (i.e. the tasks of
+             * the root project and all the subprojects).
+             *
+             * <p>If a certain project does not have a task with the specified name, no action is
+             * performed for that project.
+             *
+             * <p>This method does not guarantee that the task will be depended on either task.
+             *
+             * <p>Invocation of this method may substitute the invocation of
+             * {@link #insertAfterTask(TaskName)} or {@link #insertBeforeTask(TaskName)} if it's
+             * guaranteed that the task will receive at least one dependency. Though the fallback
+             * is never handled and there is no guarantee that the task will get into the Gradle
+             * task graph.
+             *
+             * @param target the name of the tasks, serving as "after" anchor
+             * @return the current instance of {@link Builder}
+             */
+            public Builder insertAfterAllTasks(TaskName target) {
+                checkNotNull(target, "tasks before the new one");
+                this.previousTaskOfAllprojects = target;
+                return this;
+            }
+
+            /**
+             * Specify the files or/and directories which are the input data for the new task.
+             *
+             * <p>If none of the specified file system elements are present before the task
+             * execution, the task will be marked as {@code NO-SOURCE} and skipped.
+             *
+             * <p>Read more about the Gradle incremental builds
```

Comment:
This is totally out of the blue. Please don't do that to the potential readers, that looks totally unrelated.